cirq/ops/common_gates.py

```diff
@@ -536,9 +536,9 @@ def _decompose_(self, qubits):
             yield cirq.XPowGate(global_shift=-0.25).on(q)
             return
 
-        yield Y(q)**0.25
-        yield X(q)**self._exponent
-        yield Y(q)**-0.25
+        yield YPowGate(exponent=0.25).on(q)
```

Comment:
Why the switch in notation here? Avoiding a cycle?

Reply:
Yes, exactly.